testing/subreconciler.go

```diff
@@ -238,6 +240,9 @@ func (tc *SubReconcilerTestCase) Run(t *testing.T, scheme *runtime.Scheme, facto
 	if tc.Verify != nil {
 		tc.Verify(t, result, err)
 	}
+	if tc.VerifyCtx != nil {
+		tc.VerifyCtx(t, ctx)
```

Comment:
this will allow for additional validation, but it won't suppress existing validation for stashed values.

Reply:
The two might co-exist since

> // ExpectStashedValues ensures each value is stashed. **Values in the stash that are not expected are ignored. Values in the stash that are not expected are ignored.** [...]

And so one might combine the two to achieve the desired coverage.